controllers/awscluster_controller.go

```diff
@@ -252,6 +262,44 @@ func (r *AWSClusterReconciler) reconcileDelete(ctx context.Context, clusterScope
 	return reconcile.Result{}, nil
 }
 
+func (r *AWSClusterReconciler) reconcileLoadBalancer(clusterScope *scope.ClusterScope, awsCluster *infrav1.AWSCluster) (*time.Duration, error) {
+	if annotations.Has(awsCluster, DisableLoadBalancerReconciliationAnnotation) {
+		clusterScope.Info("load balancer reconciliation disabled")
+
+		return r.checkForExternalControlPlaneLoadBalancer(clusterScope, awsCluster), nil
+	}
+
+	elbService := r.getELBService(clusterScope)
+
+	if err := elbService.ReconcileLoadbalancers(); err != nil {
+		clusterScope.Error(err, "failed to reconcile load balancer")
+		conditions.MarkFalse(awsCluster, infrav1.LoadBalancerReadyCondition, infrav1.LoadBalancerFailedReason, infrautilconditions.ErrorConditionAfterInit(clusterScope.ClusterObj()), err.Error())
+		return nil, err
+	}
+
+	if awsCluster.Status.Network.APIServerELB.DNSName == "" {
+		conditions.MarkFalse(awsCluster, infrav1.LoadBalancerReadyCondition, infrav1.WaitForDNSNameReason, clusterv1.ConditionSeverityInfo, "")
+		clusterScope.Info("Waiting on API server ELB DNS name")
+		return pointer.Duration(15 * time.Second), nil
+	}
+
+	clusterScope.Debug("looking up IP address for DNS", "dns", awsCluster.Status.Network.APIServerELB.DNSName)
```

Comment:
```suggestion
	clusterScope.Debug("Looking up IP address for DNS", "dns", awsCluster.Status.Network.APIServerELB.DNSName)
```

Reply:
That message was inherited from the previous changes:  https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/2891a57069b8365da6c601c2aa8a9ea9ad7cde7c/controllers/awscluster_controller.go#L316

We could address this renaming in a different PR to avoid too many files changes.

Reply:
Since its a minor nit change, feel free to do it in this PR itself

Reply:
Addressed with 58e62e7b0